README.md

```diff
@@ -62,6 +62,7 @@ Running the Virtual Schema requires a Java Runtime version 11 or later.
 | [Exasol Testcontainers](https://github.com/exasol/exasol-testcontainers)            | Integration test Exasol instance on Docker             | MIT License                   |
 | [Hamcrest Resultset Matcher](https://github.com/exasol/hamcrest-resultset-matcher)  | Validating JDBC resultsets                             | MIT License                   |
 | [Java Hamcrest](http://hamcrest.org/JavaHamcrest/)                                  | Checking for conditions in code via matchers           | BSD License                   |
+| [Jacoco Agent](https://www.jacoco.org/jacoco/)                                      | Collecting coverage information                        | Eclipse Public License 2.0    |
```

Comment:
should we also add a line for the `project-keeper` in the `Build Dependencies` section? as:

`| [Project Keeper Maven Plugin](https://github.com/exasol/project-keeper-maven-plugin)         | Checking project structure                             | MIT License                   |`

Reply:
Fixed.